hack/preview-template.env

```diff
@@ -100,7 +100,6 @@ export DOCKER_IO_AUTH=""
 ### pipelines-as-code-secret is created by preview.sh
 export PAC_GITHUB_APP_PRIVATE_KEY= # Base64 encoded private key of the GitHub APP
 export PAC_GITHUB_APP_ID= # Application ID
-export PAC_GITHUB_APP_WEBHOOK_SECRET= # Webhook secret of the GitHub APP
```

Comment:
why this env var was removed when it's still used? Only description comment should be updated.

Reply:
@Michkov my fault. I added it back in PR https://github.com/redhat-appstudio/infra-deployments/pull/2316